blog.html

```diff
@@ -2,24 +2,77 @@
 <html lang="en">
 	<head>
     	<meta charset="utf-8">
-    	<meta name="description" content="Your description goes here">
-    	<meta name="keywords" content="one, two, three">
+    	<meta name="description" content="Relaxr Blog">
+    	<meta name="keywords" content="Relaxr, Blog, Travel, excel, automation">
 
-		<title>Blank Template</title>
+		<title>Relaxr Blog</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800' rel='stylesheet' type='text/css'>
 
 	</head>
 	<body>
 		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+			<header class="clearfix">
+				<h1 class="logo">Relaxr</h1>
+				<nav>
+					<ul class="topnav clearfix">
+						<li><a href="#">About</a></li>
+						<li><a href="#">FAQ</a></li>
+						<li><a href="#">Team</a></li>
+						<li><a href="#">Contact Us</a></li>	
+						<li><a href="#">Blog</a></li>	
+					</ul>
+				</nav>
 			</header>
-			<footer>
-				&copy; Footer Goes Here
+			<main class="clearfix">
+				<aside class= "clearfix">
```

Comment:
No need to add the `clearfix` class to the actual elements being floated inside of a `clearfix` container, unless those elements themselves have items being floated inside of them. You only need `clearfix` on a container for floated elements.